### ISSUE TITLE
CuteSpotify on ubuntu touch always switches back to 'very loud' for every new song

### ISSUE THREAD

**Issue by KasperJanssens** [Elleo/cutespotify#13]
When I start a playlist and I put the sound on a level that's okay for my ears, every time a new song starts it goes back too very loud and I have to switch the sound back lower. Not sure whether it's an ubuntu issue or a cutespotify issue, but I'm guessing the latter (no evidence for that though).

**Comment by Bastien-** [comment]
Same thing happens to me. On an ubuntu phone, the aquaris that has just went on sale. To reproduce it I just have to launch cute spotify and hit play on any song. The volume of the song will be the max although the system can be set differently. And in the system volume settings it shows that the volume is not at max. If I move it even slightly it takes precedence again.

**Comment by Elleo** [comment]
Thanks for reporting this; I suspect it might actually be a bug in the Ubuntu Touch qtmultimedia backend, since this behaviour doesn't happen under Sailfish; I'll do some more investigation when I have a bit of free time and see if I can figure out exactly what's going on.

**Comment by tpeeters** [comment]
Yes, same problem here on the BQ Ubuntu phone. This makes the app useless because every time the next song starts I would become completely deaf when wearing headphones. I'm t1mp on freenode irc, ping me if you need me to check/verify anything on the device.

**Comment by tpeeters** [comment]
A bug in indicator-sound is being tracked here https://bugs.launchpad.net/ubuntu/+source/indicator-sound/+bug/1418210

**Comment by jenslind** [comment]
Is this fixed? This does not happen to me on BQ E5.
